docs/sources/write/front-matter/index.md

```diff
@@ -68,38 +69,65 @@ The correct way to use aliases depends on whether the project is versioned or no
 
 #### Versioned projects
 
-Do not include an `aliases` entry that refers to the initial published website directory.
-The version in the URL path can cause undesirable redirects, such as a redirect from latest content to an old version.
-Aliases must be relative and not absolute paths so that old versions do not steal redirects from "latest" content when it is moved around.
+Aliases must be relative to avoid redirecting latest content to old versions.
+This happens when a page no longer exists in latest documentation but an alias with the latest version exists in old documentation.
 
 Aliases should include a YAML comment explaining the absolute URL path that the relative path redirects.
-This helps a reviewer check that your alias works correctly.
-For example, `../path/to/alias/ # /docs/grafana/<GRAFANA VERSION>/path/to/alias/`.
+This helps a reviewer confirm that your alias works correctly.
+
+A YAML comment starts with a hash (`#`).
+
+For example, the following Markdown front matter snippet, in the file `new-url.md`, defines an alias to redirect `/docs/grafana/<GRAFANA_VERSION/original-url/`.
+The comment `# /docs/grafana/<GRAFANA_VERSION>/original-url/` indicates the absolute URL path.
+
+```
+---
+aliases:
+  - ./original-url/ # /docs/grafana/<GRAFANA_VERSION>/original-url/
+```
+
+##### Determine the relative alias
+
+To determine the relative alias, you must first understand the meaning of the current directory (`.`) and parent directory (`..`) path elements when they are used at the start of an alias.
 
-##### Examples
+The following example redirects _`SOURCE`_ to _`DESTINATION`_ using the pages:
 
-To redirect the page `/docs/grafana/latest/alerting/silences/` to `/docs/grafana/latest/alerting/manage-notifications/create-silence/`, you must add a relative alias in the source file for `/docs/grafana/latest/alerting/manage-notifications/create-silence/` containing the relative alias to `/docs/grafana/latest/alerting/silences/`.
+| Variable        | Page                                                                 |
+| --------------- | -------------------------------------------------------------------- |
+| _`SOURCE`_      | `/docs/grafana/latest/alerting/silences/`                            |
+| _`DESTINATION`_ | `/docs/grafana/latest/alerting/manage-notifications/create-silence/` |
 
-- The relative alias `./` refers to the page`/docs/grafana/latest/alerting/manage-notifications/` because that is the directory containing the page `/docs/grafana/latest/alerting/manage-notifications/create-silence/`.
-- The relative alias `../` refers to the page `/docs/grafana/latest/alerting/`.
-- The relative alias `../silences/` refers to the page `/docs/grafana/latest/alerting/silences/`.
+- The current directory element (`.`) refers to the directory containing _`DESTINATION`_.
```

Comment:
This is confusing because there is no `.` in the table.   If you're giving an example, the example needs to include the `.` .

Reply:
I'm a little confused. Are you saying that there should be a `.` entry in the table?

I was trying to use the table to better display the source and destination URL paths. The existing paragraphs look really noisy with the preformatted URL paths.

Maybe I should rewrite this example to just demonstrate the values of `.` and `..` with a concrete page and then I can later have examples for determining the redirect between _`SOURCE`_ and _`DESTINATION`_?

Reply:
The way you had it presented before with the smaller table, the text seemed to refer to the table.  But there was no current directory notation in the table, so it was confusing.  The table in your current version is less confusing in my opinion.